doc/feature/pointDataEvolution.md

```diff
@@ -0,0 +1,85 @@
+Evolution of data at a point over time
+======================================
+
+One of the features that would make the tool extremely useful for scientists, is
+the ability to obtain the evolution of the data at a specific point over time.
+For example, they could load a current report and compute the LFP for a specific
+target, and once they have their full volume, they could focus on a specific
+point of the scene (e.g. a voxel), and obtain a 2D curve showing the evolution
+of voltage at that point over time.
+
+## Requirements
+
+* Users should be able to specify the coordinates of a 3D point as input.
+* As output, users will obtain a series of values corresponding to the evolution
+  of the data at that specific point over time (e.g. one value per simulation
+  frame).
```

Comment:
There's an interesting problem to consider here. If the new function returns one sample per timestamp at the same time resolution than the simulation we can take the result as good, but if the user could request a subsampling of the data we have to think about temporal-aliasing issues. The whole point of this comment is that the spec should be clear (and not give an example) about what is the time series returned, how the metadata associated with it can be retrieved and what filtering is applied, if any.